content/en/docs/concepts/workloads/pods/pod-topology-spread-constraints.md

```diff
@@ -284,8 +284,6 @@ There are some implicit conventions worth noting here:
 
 ### Cluster-level default constraints
 
-{{< feature-state for_k8s_version="v1.19" state="beta" >}}
-
```

Comment:
Is this [what feature] moving to GA?

Reply:
Not really, this is GA since 1.19, along with the rest of the feature in this doc. Should I fix in master instead?

Reply:
I think it's _also_ OK to fix the master branch, but not wrong to fix it here.

Reply:
#24866